contents/blog/posthog-alternatives.md

```diff
@@ -0,0 +1,738 @@
+
+- "I absolutely don't like the fact that we can't extend our session retention without upgrading our plan to a much higher level."
+
+- "The costs limit our usage a bit - we've had to reduce our session count significantly because we kept exceeding the rate limit before the next cycle."
+
+Users consistently praise LogRocket's session replay tools, but other features are less developed:
+
+- "LogRocket's additional features outside session replay are harder to set up and understand. We have not been able to fully maximize our platform use or find extra value outside of session replay."
+
+- "The metrics feature is promising, but not a substitute for traditional product analytics tools."
+
+**Read more:** [In-depth PostHog vs LogRocket comparison](/blog/posthog-vs-logrocket)
+
+## Why people choose PostHog
+
+Assembly AI [switched from Mixpanel to PostHog](https://posthog.com/customers/assemblyai) because Mixpanel's pricing caused them to throttle events. Using PostHog also meant they could combine analytics and A/B testing within a single plaform: 
+
+> "At the time I was doing A/B experimentation on a different platform, then analyzing the results in Mixpanel. When Mixpanel’s pricing ramped up we looked at other platforms, including things like Segment — but PostHog, which can do both experiments and analytics in one, was clearly the winner."
+
+<BorderWrapper>
+    <Quote
+        imageSource="/images/customers/alberto.jpg"
+        size="md"
+        name="Alberto Santos"
+        title="Web & Brand Lead, AssemblyAI"
+        quote={`“PostHog helps us debug support issues, because we push errors to PostHog as events. It may not be exactly what PostHog was intended for, but it is really useful and shows how adaptable PostHog is.”`}
+    />
+</BorderWrapper>
+
+Contra chose PostHog to [replace a complex stack of separate tools](https://posthog.com/customers/contra), and eliminate data inconsistencies between them: 
+
+> “Originally we were using multiple platforms. That meant Mixpanel for analytics, LaunchDarkly for feature flags, Segment for front-end tracking and FullStory for replays. In isolation they could work well, but they were costly and complex.”
+
+<BorderWrapper>
+    <Quote
+        imageSource="/images/customers/allison.jpg"
+        size="md"
+        name="Allison Nulty"
+        title="Head of Product, Contra "
+        quote={`“A huge competitive advantage has been the ability to talk directly with PostHog engineers over Slack. We share feedback, ask questions, and make requests and always see a quick response time and thoughtful suggestions. At Contra, we apply these same principles of collaborating with our community to build our product.”`}
+    />
+</BorderWrapper>
+
+Netdata chose PostHog because they [wanted a developer-centric tool](/customers/netdata)built for startups. It also allowed them to replace Smartlook, their previous session replay too:
```

Comment:
```suggestion
Netdata chose PostHog because they [wanted a developer-centric tool](/customers/netdata) built for startups. It also enabled them to replace Smartlook, their previous session replay too:
```